pyvrp/cpp/search/SwapStar.cpp

```diff
@@ -93,6 +93,78 @@ std::pair<Cost, Route::Node *> SwapStar::getBestInsertPoint(
     return std::make_pair(deltaCost, p(V));
 }
 
+Cost SwapStar::evaluateRoute(Route::Node *V,
+                             Route::Node *U,
+                             Route::Node *UAfter,
+                             CostEvaluator const &costEvaluator)
```

Comment:
We call this a `(U, V)` pair of clients in many other operators. I think it makes sense to preserve that order here as well. Can we swap the arguments to be something like
```suggestion
Cost SwapStar::evaluateMove(Route::Node *U,
                            Route::Node *V,
                            Route::Node *remove,
                            CostEvaluator const &costEvaluator)
```
with the interpretation that we're inserting `U` after `V` (like (1, 0)-exchange), while simulateously removing `remove`. Or something similar.